app/components/floating_text_input_label/animated_input/index.tsx

```diff
@@ -238,6 +238,7 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
                     </Animated.Text>
                 }
                 <TextInput
+                    autoCapitalize={'none'}
```

Comment:
if you want this to apply to all of them, that you shouldn't want that, this prop should go below `{...props}` the reality is that you should pass this as a prop on the parent component.